connexta/eve
# 429 Error Hotfix

#### What does this PR do?
Reduces amount of api calls to slack to reduce the chance of 429 errors. Will prevent from displaying slack until everything is loaded.

#### Who is reviewing it?
@vinamartin @Kjames5269 @alchucam @mcalcote

#### How should this be tested?
Start the app in browser and open console to see if 429 errors occur and if the wallboard can handle them without crashing.

#### Screenshots

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
